### PR TITLE
v9.2.0

### PR DESCRIPTION
**Pull request by adamraine**

### CODE REVIEW

File: docs/releasing.md [hunk 1]
```diff
@@ -50,9 +50,9 @@ yarn build-all
 
 # Verify the viewer will work.
 yarn serve-viewer
-# Works with v4 report? http://localhost:8000/viewer?gist=7251f9eba409f385e4c0424515fe8009
```
Comment (adamraine) [inline docs/releasing.md:53]: v4 report wasn't working. I don't think we need to support this anymore.

Comment (brendankenny) [reply docs/releasing.md:53]: Ideally we can avoid contributing to URL rot. https://googlechrome.github.io/lighthouse/viewer/?gist=7251f9eba409f385e4c0424515fe8009 is working (currently at 9.1.0), so it was a change in this release?

Comment (brendankenny) [reply docs/releasing.md:53]: actually, https://lighthouse-e6g7mely5-googlechrome.vercel.app/gh-pages/viewer/?gist=7251f9eba409f385e4c0424515fe8009 also seems to work from this deploy. Some local issue?

Comment (connorjclark) [reply docs/releasing.md:53]: what's the error? seems good to me

File: changelog.md [hunk 2]
```diff
@@ -1,3 +1,51 @@
...
+* move no-unload-listeners to perf category ([#13497](https://github.com/GoogleChrome/lighthouse/pull/13497))
+
+## Core
+
+* font-size: fix CSS selector regex ([#13455](https://github.com/GoogleChrome/lighthouse/pull/13455))
+* fcp: handle negative request `endTime` ([#13452](https://github.com/GoogleChrome/lighthouse/pull/13452))
+* legacy-javascript: latest polyfilling, drop some signals ([#13482](https://github.com/GoogleChrome/lighthouse/pull/13482))
+* cache fetch to avoid bad polyfills ([#13476](https://github.com/GoogleChrome/lighthouse/pull/13476))
+* computed artifact keys ([#13430](https://github.com/GoogleChrome/lighthouse/pull/13430))
+* legacy-javascript: remove Object.getOwnPropertyNames signal ([#13442](https://github.com/GoogleChrome/lighthouse/pull/13442))
```
Comment (connorjclark) [inline changelog.md:18]: combine with above entry

File: changelog.md [hunk 3]
```diff
@@ -1,3 +1,51 @@
...
+
+## ⛏️👷 Fraggle Rock
+
+  Support for auditing user flows ([#11313](https://github.com/GoogleChrome/lighthouse/issues/11313))
+
+* fr: compute timespan saving with observed throughput ([#13478](https://github.com/GoogleChrome/lighthouse/pull/13478))
```
Comment (connorjclark) [inline changelog.md:24]: remove fr:

File: changelog.md [hunk 4]
```diff
@@ -1,3 +1,51 @@
...
+* fr: option to skip about:blank jumps ([#13375](https://github.com/GoogleChrome/lighthouse/pull/13375))
+
+## Report
+
+* psi: update styles for tighter fit in narrow viewports ([#13355](https://github.com/GoogleChrome/lighthouse/pull/13355))
+* port axe test to pptr ([#13450](https://github.com/GoogleChrome/lighthouse/pull/13450))
```
Comment (connorjclark) [inline changelog.md:30]: move to tests

File: changelog.md [hunk 5]
```diff
@@ -1,3 +1,51 @@
...
+
+## Notable Changes
+
+* move no-unload-listeners to perf category ([#13497](https://github.com/GoogleChrome/lighthouse/pull/13497))
+
+## Core
+
+* font-size: fix CSS selector regex ([#13455](https://github.com/GoogleChrome/lighthouse/pull/13455))
+* fcp: handle negative request `endTime` ([#13452](https://github.com/GoogleChrome/lighthouse/pull/13452))
+* legacy-javascript: latest polyfilling, drop some signals ([#13482](https://github.com/GoogleChrome/lighthouse/pull/13482))
```
Comment (brendankenny) [inline changelog.md]: maybe

```suggestion
* legacy-javascript: update polyfilling, drop some signals ([#13482](https://github.com/GoogleChrome/lighthouse/pull/13482))
```

File: changelog.md [hunk 6]
```diff
@@ -1,3 +1,51 @@
...
+* cache fetch to avoid bad polyfills ([#13476](https://github.com/GoogleChrome/lighthouse/pull/13476))
+* computed artifact keys ([#13430](https://github.com/GoogleChrome/lighthouse/pull/13430))
```
Comment (brendankenny) [inline changelog.md:17]: our commit message hygiene is slipping. Needs a verb :P

File: docs/releasing.md (hunk 1 above)
Comment (adamraine) [reply docs/releasing.md:53]: Ope false alarm. I thought it didn't generate the opportunities or diagnostics section. Turns out it just passed everything and I missed the passed audit section 🙃 .

File: third-party/chromium-synchronization/inspector-issueAdded-types-test.js [hunk 7]
```diff
@@ -36,6 +36,7 @@ describe('issueAdded types', () => {
 Array [
   "attributionReportingIssueDetails",
   "blockedByResponseIssueDetails",
+  "clientHintIssueDetails",
```
Comment (adamraine) [inline third-party/chromium-synchronization/inspector-issueAdded-types-test.js:39]: Is this a bad time?

**Review by connorjclark — APPROVED** [review]

**Pull request merged**